src/utils/api/cron.helper.ts

```diff
@@ -44,6 +45,15 @@ export async function getData(round: number) {
   );
   // const priceFbeets = await getTokenPrice(end, "0xfcef8a994209d6916eb2c86cdd2afd60aa6f54b1");
 
+  let pricePerVp = priceFbeets;
```

Comment:
Yes, that might do it. I tried ternary operator first, but discarded it, because I did not like it. 

if I do it the way you suggested, shouldn't there be a `()` at the end to actually run the anonymous function?